client/post-editor/editor-notice/index.jsx

```diff
@@ -122,62 +122,44 @@ export class EditorNotice extends Component {
 				}
 
 				if ( 'page' === type ) {
-					return translate( 'Page published on {{siteLink/}}! {{a}}Add another page{{/a}}', {
-						components: {
-							siteLink: (
-								<a
-									href={ site.URL }
-									target="_blank"
-									rel="noopener noreferrer"
-									onClick={ this.handlePillExternalClick }
-								>
-									{ site.title }
-								</a>
-							),
-							a: (
-								<a href={ `/page/${ site.slug }` } onClick={ this.handlePillAddPagePromptClick } />
-							),
-						},
-						comment:
-							'Editor: Message displayed when a page is published, with a link to the site it was published on.',
-					} );
+					return translate(
+						'Page published! {{postLink}}Visit post{{/postLink}} or {{newPageLink}}add another page{{/newPageLink}}.',
+						{
+							components: {
+								postLink: <a href={ postUrl } onClick={ this.handlePillExternalClick } />,
+								newPageLink: (
+									<a
+										href={ `/page/${ site.slug }` }
+										onClick={ this.handlePillAddPagePromptClick }
+									/>
+								),
+							},
+							comment:
+								'Editor: Message displayed when a page is published, with a link to the page and a link to add a new page.',
+						}
+					);
 				}
 
 				if ( 'post' !== type && typeLabel ) {
-					return translate( '%(typeLabel)s published on {{siteLink/}}!', {
-						args: { typeLabel },
-						components: {
-							siteLink: (
-								<a
-									href={ site.URL }
-									target="_blank"
-									rel="noopener noreferrer"
-									onClick={ this.handlePillExternalClick }
-								>
-									{ site.title }
-								</a>
-							),
-						},
-						comment:
-							'Editor: Message displayed when a post of a custom type is published, with a link to the site it was published on.',
-					} );
+					return translate(
+						'%(typeLabel)s published! {{postLink}}Visit %(typeLabel)s{{/postLink}}.',
+						{
+							args: { typeLabel },
+							components: {
+								postLink: <a href={ postUrl } onClick={ this.handlePillExternalClick } />,
+							},
+							comment:
+								'Editor: Message displayed when a post of a custom type is published, with a link to the published post.',
+						}
+					);
 				}
 
-				return translate( 'Post published on {{siteLink/}}!', {
+				return translate( 'Post published! {{postLink}}Visit post{{/postLink}}.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Post updated! {{postLink}}Visit post{{/postLink}}.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).